### ISSUE TITLE
Any chance for C/C++ support via gdb or lldb?

### ISSUE THREAD

**Issue by ell1e** [atom-community/atom-ide-debugger#12]
It seems like currently all debugger packages for C/C++ for Atom are dead, sadly. Will this one eventually add C/C++ support to solve this issue, via lldb or gdb? That would make Atom way more feasible for system level development again.

**Comment by aminya** [comment]
@ell1e Yes, I will add support for C/C++. I am looking to make a full-fledged C/C++ package to support all the cases.

**Comment by ell1e** [comment]
If I may suggest, it seems like clangd might be a great option for the non-debugging parts like auto complete, go to declaration, etc. Not only does clangd work very well for me in VS Code with its useful output, but it also has a  versatile config for extra options like easily changing flags only for some files, etc. in a `.clangd` file that can be put into the project root. While the same can be achieved with `cmake` and `compile_commands.json`, this needs a user to run an extra tool manually before getting helpful output, while with clangd + `.clangd` it will "just work" when opening the project in any IDE with any clangd-based diagnostics plugin. Very cool.
